deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -221,23 +221,63 @@ cephClusterSpec:
   #   # These labels can be passed as LabelSelector to Prometheus
   #   monitoring:
 
-  # resources:
-  #   # The requests and limits set here, allow the mgr pod to use half of one CPU core and 1 gigabyte of memory
-  #   mgr:
-  #     limits:
-  #       cpu: "500m"
-  #       memory: "1024Mi"
-  #     requests:
-  #       cpu: "500m"
-  #       memory: "1024Mi"
-  #   # The above example requests/limits can also be added to the other components
-  #   mon:
-  #   osd:
-  #   prepareosd:
-  #   mgr-sidecar:
-  #   crashcollector:
-  #   logcollector:
-  #   cleanup:
+  resources:
+    mgr:
+      limits:
+        cpu: "500m"
+        memory: "512Mi"
+      requests:
+        cpu: "500m"
+        memory: "512Mi"
+    mon:
+      limits:
+        cpu: "2000m"
+        memory: "2048Mi"
+      requests:
+        cpu: "2000m"
+        memory: "2048Mi"
+    osd:
+      limits:
+        cpu: "1000m"
+        memory: "4096Mi"
+      requests:
+        cpu: "1000m"
+        memory: "4096Mi"
+    prepareosd:
+      limits:
+        cpu: "500m"
+        memory: "50Mi"
+      requests:
+        cpu: "500m"
+        memory: "50Mi"
+    mgr-sidecar:
+      limits:
+        cpu: "500m"
+        memory: "100Mi"
+      requests:
+        cpu: "500m"
```

The mgr-sidecar, crashcollector, and logcollector are really low-cpu. How about reducing them to 1/10th?
```suggestion
        cpu: "100m"
```